src/main/java/seedu/cafectrl/CafeCtrl.java

```diff
@@ -35,6 +35,7 @@ private CafeCtrl() {
         this.pantry = this.storage.loadPantryStock();
         this.sales = this.storage.loadOrderList(menu);
         this.currentDate = new CurrentDate(sales);
+        assert !sales.getOrderLists().isEmpty();
```

Comment:
could you perhaps add a newline before this statement to show clear sections for set up and testing

Reply:
Also, would this assert statement get triggered whenever the application is launch for the first time?

Reply:
oh sorry misunderstood that the sales is a array of order list not an array of orders